input/blog/2021-02-17-cake-nuget-packages--identity-and-trust.md

```diff
@@ -0,0 +1,28 @@
+﻿---
+title: "Cake NuGet Packages: Identity and Trust"
+category: Announcement
+author: augustoproiete
+---
+The NuGet team at Microsoft introduced a concept known as "[Package ID prefix reservation](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation)" for packages published on [nuget.org](https://www.nuget.org), which allows developers to reserve particular package ID prefixes with a specific account/owner and offers a number of features that are [described in detail on Microsoft's docs](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation).
+
+One of these features is that packages with a reserved prefix include a visual indicator for NuGet package consumers to identify that the packages they are consuming originate from package owners that have reserved the prefix.
+
+![Cake package visual indicator on NuGet Gallery](media/2021-02-17-cake-nuget-packages--identity-and-trust/cake-package-visual-indicator-nuget-screenshot.png)
```

Comment:
Image is not shown for me when I build locally and go to http://localhost:5080/blog/2021/02/cake-nuget-packages--identity-and-trust. It is shown though on http://localhost:5080/blog/.

On blog detail page image link is http://localhost:5080/blog/2021/02/media/2021-02-17-cake-nuget-packages--identity-and-trust/cake-package-visual-indicator-nuget-screenshot.png instead of http://localhost:5080/blog/media/2021-02-17-cake-nuget-packages--identity-and-trust/cake-package-visual-indicator-nuget-screenshot.png

Reply:
Good catch. Fixed it